libsoftwarecontainer/src/container.cpp

```diff
@@ -57,15 +57,15 @@ void Container::init_lxc()
     }
 }
 
-Container::Container(const std::string &id, const std::string &name, const std::string &configFile,
+Container::Container(const std::string id, const std::string &configFile,
         const std::string &containerRoot, bool enableWriteBuffer, int shutdownTimeout) :
     m_configFile(configFile),
     m_id(id),
-    m_name(name),
     m_containerRoot(containerRoot),
     m_enableWriteBuffer(enableWriteBuffer),
     m_shutdownTimeout(shutdownTimeout)
 {
+    log_debug() << "Container constructed with " + id;
```

Comment:
I'd << operator instead of + operator for style reasons, not sure what you think.